### PR TITLE
Dealing with too much logMessages in CSC detailed view

### PR DESCRIPTION
**Pull request by fdelcampo**
Set a slice for the arrays logMessageData and errorCodeData

### CODE REVIEW

File: love/src/components/EventLog/EventLog.container.jsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ const mapDispatchToProps = (dispatch) => {
 
 const mapStateToProps = (state, ownProps) => {
   const cscList = Object.values(CSCSummaryHierarchy).flatMap((realm) => {
-    // console.log(realm);
+    //console.log(realm);
```
Comment (sebastian-aranda) [inline love/src/components/EventLog/EventLog.container.jsx:105]: This should be removed

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Thanks @fdelcampo , I only left a comment regarding a console.log

File: love/src/components/EventLog/EventLog.container.jsx (hunk 1 above)
Comment (fdelcampo) [reply love/src/components/EventLog/EventLog.container.jsx:105]: yes, in other commit

**Review by sebastian-aranda — APPROVED** [review]
Thanks @fdelcampo this is ready to be merged

**Pull request merged**